pkg/knowledge_base2/emitter.go

```diff
@@ -90,32 +90,31 @@ func ConsumeFromResource(consumer, emitter *construct.Resource, ctx DynamicConte
 					addErr(consume, emit, err)
 					continue
 				}
-				val, err = sanitizeForConsumption(ctx, resource, consumeTmpl.GetProperty(consume.PropertyPath), val)
+				pval, err := resource.GetProperty(consume.PropertyPath)
```

Comment:
nit: why did this move up? `pval` isn't used until after converting & sanitizing so it can be done later